canbusload.c

```diff
@@ -243,14 +246,16 @@ static void printstats(int signo)
 int main(int argc, char **argv)
 {
 	fd_set rdfs;
-	int s[MAXSOCK];
-
+	int s;
 	int opt;
 	char *ptr, *nptr;
 	struct sockaddr_can addr;
 	struct canfd_frame frame;
 	int nbytes, i;
-	struct ifreq ifr;
+
+	int have_anydev = 0;
```

Comment:
`bool`?

Reply:
no. This is needed for an offset calculation later.

Reply:
I currently have a setup with 16 interfaces on my desk - partly with CAN FD.
And it turned out that the canbusload tool needed some love ;-)